csvqb/csvqb/tests/unit/writers/test_skoscodelistwriter.py

```diff
@@ -0,0 +1,92 @@
+import json
+
+import pytest
+from typing import Dict, Any
+from rdflib import RDFS, Literal, URIRef, Graph
+
+from csvqb.models.cube import *
+from csvqb.models.cube.csvqb.components.arbitraryrdf import (
+    RdfSerialisationHint,
+    TripleFragment,
+)
+from csvqb.writers.skoscodelistwriter import SkosCodeListWriter
+
+
+basic_code_list = NewQbCodeList(
+    CatalogMetadata("Some CodeList"),
+    [
+        NewQbConcept(
+            "First Concept",
+            code="1st-concept",
+            description="This is the first concept.",
+        ),
+        NewQbConcept("Second Concept", parent_code="1st-concept", sort_order=20),
+    ],
+)
+
+
+def test_code_list_data_mapping():
```

Comment:
Got confused then realised this is a renamed file. I was about to ask you to stay in scope.

Reply:
Well, it's a bit more than that... the test file hadn't been migrated over to the pytest style... and I'm not 100% sure it was being run before.